src/module/apps/item-attacher.ts

```diff
@@ -6,6 +6,12 @@ import { ErrorPF2e } from "@util";
 
 /** A prompt for the user to select an item to receive an attachment */
 class ItemAttacher<TItem extends PhysicalItemPF2e> extends PickAThingPrompt<TItem, PhysicalItemPF2e> {
+    static override DEFAULT_OPTIONS: DeepPartial<fa.ApplicationConfiguration> = {
+        window: {
+            contentClasses: ["item-attacher"],
+        },
+    };
```

Comment:
We only ever want one of these open, so just give it an id of "item-attacher"

Reply:
Done